### DISCUSSION TITLE
v-model 语法糖是否能简洁为 1 个字符

### DISCUSSION THREAD

**Discussion by natholdallas** [vuejs/rfcs#661] [General]
我知道v-model本质是一个语法糖,emit 和 props 的再次封装, 虽然很优雅,但我在 dom 中使用的时候, 如果 v-model 多了起来, dom很容易换行, 请问以后会支持 v-model 可以简洁为 1个字符吗? 比如 $, % ^ &, 任何字符都可以只要符合语义(抽象化也可以)

**Comment by edison1105** [comment]
see https://vue-macros.dev/macros/short-vmodel.html

**Reply by natholdallas** [reply]
thanks so much, you are my hero
